packages/fluentui/react-northstar/src/components/Attachment/AttachmentAction.tsx

```diff
@@ -1,13 +1,29 @@
 import { buttonBehavior } from '@fluentui/accessibility';
+import * as customPropTypes from '@fluentui/react-proptypes';
+import * as PropTypes from 'prop-types';
 import { compose, ComponentWithAs } from '@fluentui/react-bindings';
 
-import { createShorthandFactory, ShorthandFactory } from '../../utils';
+import { commonPropTypes, ShorthandConfig } from '../../utils';
 import Button, { ButtonProps, ButtonStylesProps } from '../Button/Button';
 
 interface AttachmentActionOwnProps {}
-export interface AttachmentActionProps extends AttachmentActionOwnProps, ButtonProps {}
+export interface AttachmentActionProps extends AttachmentActionOwnProps, ButtonProps {
```

Comment:
Should we use `Pick` instead of `never`?

Reply:
In my opinion was easier to just turn off the ones I don't want to use instead of pick everything else, as I am pretty sure it's easier to forget at least some of the common props...